frontend/src/components/MeetingRoom/ArchivingToggle/ArchivingToggle.tsx

```diff
@@ -78,6 +80,10 @@ const ArchivingToggle = (): ReactElement => {
               style={{ color: `${isRecording ? 'rgb(239 68 68)' : 'white'}` }}
             />
           }
+          sx={{
+            width: isSmallViewport ? '42px' : '48px',
+            height: isSmallViewport ? '42px' : '48px',
```

Comment:
Note: these were needed here and in other spots to make sure that these buttons are smaller in the tiny tab ™️ devices (such as iPhone SE). I could not use `isSmallViewPort` directly on `ToolbarButton` since we have components that are dependent on it such as the toolbar overflow menu opener and the hang up button which we want to remain the same even on the tiny tab ™️ devices.

Reply:
Could we add a prop to the `ToolbarButton`? Maybe `isSmallViewport` or `isOverflowButton` so we can apply the changes inside the component instead of adding `sx` props for all of these?

Reply:
great idea, added!